bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/utils/Utils.java

```diff
@@ -0,0 +1,15 @@
+package org.codechecker.eclipse.plugin.utils;
+
+public class Utils {
+
+    /**
+     * Escapes the ' character in a String. Should be used on all paths Strings.
+     * @param path The input path.
+     * @return Will be the same String if no ' to be found. If there are ''s in the String
+     * each of the will be replaced with this: '\''.
+     * {@link https://unix.stackexchange.com/a/357932}
+     */
+    public static String esc(String path) {
+        return path.replace("'", "'\\''");
```

Comment:
There are some [string escaping methods](https://commons.apache.org/proper/commons-text/javadocs/api-release/org/apache/commons/text/StringEscapeUtils.html) could we use them? Should we escape other characters too?

Reply:
This section was replaced with the path substitution map.